src/components/Home/HomeSection.jsx

```diff
@@ -29,9 +31,10 @@ const HomeSection = ({
   button,
   href,
   label,
-  isReallySmallScreen
+  isReallySmallScreen,
+  noMargin
```

Comment:
nit: I'd probably opt for a positive and for a boolean-esque naming (`hasMargin`) over `noMargin`. `noMargin={false}` or `{noMargin ? null : 8}` is slightly harder to grok than `hasMargin={true}` or `{hasMargin ? 8 : null}`

Reply:
I can switch... I think my initial approach was for purposes of less code, but this suggestion makes sense & matches what I've seen from component libraries.